packages/upload-api/src/store/get.js

```diff
@@ -14,16 +15,9 @@ export function storeGetProvider(context) {
     }
     const space = Server.DID.parse(capability.with).did()
     const res = await context.storeTable.get(space, link)
-    if (!res) {
-      return {
-        error: {
-          name: 'StoreItemNotFound',
-          message: 'Store item not found',
-        },
-      }
-    }
-    return {
-      ok: res,
+    if (res.error && res.error.name === 'RecordNotFound') {
+      return Server.error(new StoreItemNotFound(space, link))
```

Comment:
Shouldn't this be wrapped into `error` like before?

`context.storeTable.get(space, link)` returns a Result<A, B>, but `ucanto` error is not such format https://github.com/web3-storage/ucanto/blob/main/packages/server/src/handler.js#L268 ?

Therefore, this is inconsistent with return for res, as well as other files like `list.js`

Reply:
so it is not consistent with server.fail https://github.com/web3-storage/ucanto/blob/main/packages/core/src/result.js#L58 too

Reply:
`Server.error(new Error('boom'))` => `{ error: new Error('boom') }`